core/src/test/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8StressTest.java

```diff
@@ -0,0 +1,96 @@
+package org.infinispan.commons.util.concurrent.jdk8backported;
+
+import org.infinispan.commons.equivalence.AnyEquivalence;
+import org.infinispan.commons.util.concurrent.jdk8backported.BoundedEquivalentConcurrentHashMapV8.Eviction;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.AssertJUnit.fail;
+import static org.testng.AssertJUnit.assertNotNull;
+
+/**
+* @author William Burns
+* @since 7.1
+*/
+@Test(groups = "stress", testName = "util.concurrent.BoundedEquivalentConcurrentHashMapV8StressTest")
+public class BoundedEquivalentConcurrentHashMapV8StressTest extends AbstractInfinispanTest {
+   protected static final int COUNT = 10_000_000;
+   
+   protected void testRemovePerformance(int count, Map<Integer, Integer> bchm,
+         String evictionName) {
+      long startIncludePut = System.currentTimeMillis();
+
+      int j = 0;
+      long firstHalfNano = System.nanoTime();
+      // fill the cache
+      for (; j < count / 2; j++)
+         bchm.put(j, j);
+      firstHalfNano = System.nanoTime() - firstHalfNano;
+
+      long secondHalfNano = System.nanoTime();
+      for (; j < count; j++)
+         bchm.put(j, j);
+      secondHalfNano = System.nanoTime() - secondHalfNano;
+      
+      // force a single cache hit (so that accessQueue has a head item)
+      bchm.get(0);
+
+      // remove items
+      long start = System.currentTimeMillis();
+      long getNano = 0;
+      long removeNano = 0;
+      for (int i = 1; i < count; i++)
+      {
+         long getBegin = System.nanoTime();
+         assertNotNull(bchm.get(i));
+         getNano += System.nanoTime() - getBegin;
+         long removeBegin = System.nanoTime();
+         bchm.remove(i);
+         removeNano += System.nanoTime() - removeBegin;
+
+         // Only check every 100
+         if (i % 100 == 0) {
+            if (System.currentTimeMillis() - start > TimeUnit.SECONDS.toMillis(50))
+               fail(evictionName + ": removing " + count + " entries takes more than 50 seconds!");
+         }
+      }
+      System.out.println("BCHM Stress Test " + evictionName + " took " + 
```

Comment:
This one is just for testing purposes to compare performance.  This is just to differentiate between old and new CHMV8.  The system out is to make sure it is printed instead of logger.